kafkacat.c

```diff
@@ -78,20 +79,20 @@ int part_eof_thres = 0;
 
 
 /**
- * Fatal error: print error and exit
+ * Print error and exit if needed
  */
-void RD_NORETURN fatal0 (const char *func, int line,
```

Comment:
I don't have any warning when compiling on ubuntu 16.04, but you now have fatal0 and error0